File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/02_compito_java_2015_02_02/Libro.java
public class Libro {

private String titolo;
private String autori;
private int prezzo;

public Libro(String tit, String a, int pr){
	titolo = tit;
	autori = a;
	prezzo = pr;
}

public int getPrezzo(){
	return this.prezzo;
}

}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/02_compito_java_20150202/Libro.py
class Libro:
    def __init__(self, titolo, autori, prezzo):
        self._titolo = titolo
        self._autori = autori
        self._prezzo = prezzo
        self._copie_vendute = 1

    def aggiorna_numero_copie(self):
        self._copie_vendute += 1

    def get_titolo(self):
        return self._titolo

    def __str__(self):
        return self._titolo + " " + str(self._copie_vendute)
